### PR TITLE
Emergency fleet bump for qa

### PR DESCRIPTION
**Pull request by nicholasSUSE**
This is an emergency merge for unblocking QA efforts. 

## Problem
@manno QA Team was blocked by the lack of this bump at `dev-v2.7.11`, it was only present at `dev-v2.7`branch. 

Meanwhile `release/v2.7.11` branch on `rancher` project had this modification on the Dockerfile:
https://github.com/rancher/rancher/blob/release/v2.7.11/package/Dockerfile#L60

## Solution

I decided not to cherry-pick so I tried following the same steps as this PR:
https://github.com/rancher/charts/pull/3346

I will be merging this, in case of problems I may revert it.

### CODE REVIEW

File: charts/fleet/102.2.2+up0.8.2-rc.3/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+annotations:
+  catalog.cattle.io/auto-install: fleet-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.27.0-0'
```
Comment (diogoasouza) [inline charts/fleet/102.2.2+up0.8.2-rc.3/Chart.yaml]: Shouldnt this include 1.27?

**Review by diogoasouza — CHANGES_REQUESTED** [review]

File: charts/fleet/102.2.2+up0.8.2-rc.3/Chart.yaml (hunk 1 above)
Comment (adamkpickering) [reply charts/fleet/102.2.2+up0.8.2-rc.3/Chart.yaml]: Might that be a separate change from this PR? AFAICT this is just about unblocking QA.

File: packages/fleet/fleet/package.yaml [hunk 2]
```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/fleet/releases/download/v0.8.1/fleet-0.8.1.tgz
-version: 102.2.1
+url: https://github.com/rancher/fleet/releases/download/v0.8.2-rc.3/fleet-0.8.2-rc.3.tgz
```
Comment (adamkpickering) [inline packages/fleet/fleet/package.yaml:1]: This means that we would be providing a release candidate version of fleet in an official helm chart once we merge `dev-v2.7.11` to `release-v2.7`. Seems fishy to me. Why does QA need us to package a helm chart that contains an RC? Shouldn't we only be packaging full releases?

**Review by adamkpickering — CHANGES_REQUESTED** [review]

File: packages/fleet/fleet/package.yaml (hunk 2 above)
Comment (nicholasSUSE) [reply packages/fleet/fleet/package.yaml:1]: I hope this is temporary, Fleet team seems to have updated the Dockerfile on Rancher to point to this version but they merged this version on dev-v2.7 branch only.

File: charts/fleet/102.2.2+up0.8.2-rc.3/Chart.yaml (hunk 1 above)
Comment (nicholasSUSE) [reply charts/fleet/102.2.2+up0.8.2-rc.3/Chart.yaml]: This was my previous mistake, this was solved on the new push

File: packages/fleet/fleet-agent/generated-changes/patch/values.yaml.patch [hunk 3]
```diff
@@ -0,0 +1,11 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -1,7 +1,7 @@
+ image:
+   os: "windows,linux"
+   repository: rancher/fleet-agent
+-  tag: v0.8.2-rc.3
++  tag: v0.8.1
```
Comment (adamkpickering) [inline packages/fleet/fleet-agent/generated-changes/patch/values.yaml.patch]: This doesn't look right to me. Also, this patch doesn't exist on the `dev-v2.7` branch.

**Review by adamkpickering — CHANGES_REQUESTED** [review]

**Review by diogoasouza — APPROVED** [review]

File: packages/fleet/fleet/generated-changes/patch/values.yaml.patch [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+ image:
+   repository: rancher/fleet
+-  tag: v0.8.2-rc.3
++  tag: v0.8.1
+   imagePullPolicy: IfNotPresent
+ 
+ agentImage:
+   repository: rancher/fleet-agent
+-  tag: v0.8.2-rc.3
++  tag: v0.8.1
```
Comment (diogoasouza) [inline packages/fleet/fleet/generated-changes/patch/values.yaml.patch]: Is this right?

**Review by diogoasouza — CHANGES_REQUESTED** [review]

File: packages/fleet/fleet-agent/generated-changes/overlay/Chart.yaml.orig [hunk 5]
```diff
@@ -0,0 +1,13 @@
+annotations:
```
Comment (adamkpickering) [inline packages/fleet/fleet-agent/generated-changes/overlay/Chart.yaml.orig]: I want to highlight that in the `dev-v2.7` branch this is a patch, not an overlay. But I *think* this is fine.

File: packages/fleet/fleet/generated-changes/overlay/Chart.yaml.orig [hunk 6]
```diff
@@ -0,0 +1,20 @@
+annotations:
```
Comment (adamkpickering) [inline packages/fleet/fleet/generated-changes/overlay/Chart.yaml.orig]: Same comment: this is a patch in the `dev-v2.7` branch.

**Review by adamkpickering — APPROVED** [review]
Looks reasonable to me.

**Review by diogoasouza — APPROVED** [review]

**Comment by nicholasSUSE** [comment]
## Validation steps
  - Ensure all container images have repository and tag on the same level to ensure that all container images are included in rancher-images.txt which are used by airgap customers.
  <pre>
  Ex:-
    longhorn-controller:
      repository: rancher/hardened-sriov-cni
      tag: v2.6.3-build20230913
  </pre>
  - Add a 👍 (thumbs up) reaction to this comment once done. CI won't pass without this reaction to the github-action bot's latest validation comment.
  - Approve the PR to run the CI check.

**Comment by nicholasSUSE** [comment]
## Validation steps
  - Ensure all container images have repository and tag on the same level to ensure that all container images are included in rancher-images.txt which are used by airgap customers.
  <pre>
  Ex:-
    longhorn-controller:
      repository: rancher/hardened-sriov-cni
      tag: v2.6.3-build20230913
  </pre>
  - Add a 👍 (thumbs up) reaction to this comment once done. CI won't pass without this reaction to the github-action bot's latest validation comment.
  - Approve the PR to run the CI check.

**Pull request merged**